<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Blinduino</class>
 <widget class="QMainWindow" name="Blinduino">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color:rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>775</width>
     <height>330</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="220,110" columnstretch="300,475">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="colDerecha" stretch="130,60">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="descriptionModule">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #349295;
padding:5px;
font: 26pt;
color:white;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
margin-left:20px;
margin-right:20px;
margin-top:20px;</string>
          </property>
          <property name="text">
           <string>DESCRIPCIÓN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pinModule">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #006D70;
font:24pt;
color:white;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
margin-left:20px;
margin-right:20px;
margin-bottom:20px;</string>
          </property>
          <property name="text">
           <string>PINES</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="colIzquierda" stretch="130,60">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="nameModule">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 28pt;
background-color: #349295;
color:white;
border-top-left-radius: 10px;
border-top-right-radius: 10px;
margin-left:20px;
margin-right:20px;
margin-top:20px;</string>
          </property>
          <property name="text">
           <string>NOMBRE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="idModule">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="accessibleName">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #006D70;
font:28pt;
color:white;
border-bottom-left-radius: 10px;
border-bottom-right-radius: 10px;
margin-left:20px;
margin-right:20px;
margin-bottom:20px;</string>
          </property>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="pinLabel">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <property name="accessibleName">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #349295;
font:28pt;
border-radius: 10px;
color:white;
margin-left:20px;
margin-right:20px;
margin-top: 15%;
margin-bottom: 15%;</string>
        </property>
        <property name="text">
         <string>PIN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="detectButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="accessibleName">
         <string>Detectar módulos </string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: #349295;
color:white;
font: 28pt;
border-radius: 10px;
margin-left: 20px;
margin-right: 20px;
margin-top: 15%;
margin-bottom: 15%;
}
QPushButton:pressed {
	color:black;
    background-color: #cccccc;
    border-style: inset;
}
</string>
        </property>
        <property name="text">
         <string>DETECTAR</string>
        </property>
        <property name="shortcut">
         <string>Space</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="4">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="refreshButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
border-image:url(refreshButton.png);
background-position:center;
background-size:cover;
}

QPushButton:pressed {
border-image:url(refreshButtonClicked.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid #000;
font: 20pt;
color:white;

border-top-left-radius: 10px;
border-bottom-left-radius: 10px;

background: #006D70;
margin-left:20px;
margin-bottom:20px;
margin-top:20px;</string>
        </property>
        <property name="text">
         <string>ESTADO</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="statusLog">
        <property name="styleSheet">
         <string notr="true">margin-right:20px;
margin-bottom:20px;
margin-top:20px;
padding-left:10px;

font: 7pt;

border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
border-top: 1px solid #000;
border-bottom: 1px solid #000;
border-right: 1px solid #000;</string>
        </property>
        <property name="text">
         <string>Todo se ha realizado correctamente en la aplicación</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameModule</tabstop>
  <tabstop>idModule</tabstop>
  <tabstop>descriptionModule</tabstop>
  <tabstop>pinModule</tabstop>
  <tabstop>pinLabel</tabstop>
  <tabstop>detectButton</tabstop>
 </tabstops>
 <resources>
  <include location="blinduinoResources.qrc"/>
 </resources>
 <connections/>
</ui>
